### ISSUE TITLE
How can I use it? Now..

### ISSUE THREAD

**Issue by dan1-ru** [wei3hua2/singularity-js#1]
Hello. I try to install your lib repeatedly.

I am creating a new project, cloning your repository as a submodule.

I make the installation, create the "index.js" file in the root of the project with example code. 
Also I require "web3", here my deps:

"dependencies": {
  "singnet-js": "file:singularity-js",
  "web3": "^1.0.0-beta.48"
}

But in singnet-js folder from node_modules "dist" folder are empty and necessary files are missing. I try to run scripts from package.json, but can only build ts-files after installing "execution-time" module, after I run "npm run build" I receive this errors:

![Screenshot from 2019-03-19 23-37-28](https://user-images.githubusercontent.com/7466036/54632797-b1d10880-4aa0-11e9-9e87-f86c2a21b47a.png)

Before that:
![Screenshot from 2019-03-19 23-36-53](https://user-images.githubusercontent.com/7466036/54632866-cc0ae680-4aa0-11e9-8c87-945b6fec37f6.png)

But I already tried to launch this solution without webpack, there were also some errors there, but they may disappear if will I installed it correctly.

I know that you are preparing a package for publication in npm. I have no doubt that everything will be installed correctly. But maybe you can provide already now the weback-transpiled file of your library to try to make a browser application?

Thank you, James! Best regards.

**Comment by wei3hua2** [comment]
Hi @dan1-ru , my first issue logged. Lol. Really appreciate your interest.

Previously I was still experimenting with different build approach, which is why it can be a little messy.

I have created a new branch for this issue and make it the top priority.

I have created a project to track the initial release as well.
[Initial release](https://github.com/wei3hua2/singularity-js/projects/1)

For this branch, I will create a consistent build approach that will work for both javascript and typescript. Clean up and refactor the existing build, which is quite confusing.
